### DISCUSSION TITLE
actions autocompletion

### DISCUSSION THREAD

**Discussion by tanguyantoine** [cschleiden/vscode-github-actions#143] [Ideas]
That would be really useful if we could have some sort of autocompletion for well-known actions (like: `actions/checkout@v3`)

alternatively set a list of preferred actions so that we can re-use than quickly across every projects

Thank you for this extension which is priceless 🙏

**Comment by cschleiden** [comment]
That's something I've wanted to build for a while. Unfortunately,  we currently don't have a public API to find Actions in the Marketplace.

> alternatively set a list of preferred actions so that we can re-use than quickly across every projects

That's an interesting idea and might work as a work-around. Would have to figure out at which level and how to share this list.

**Reply by tanguyantoine** [reply]
> That's an interesting idea and might work as a work-around. Would have to figure out at which level and how to share this list.

Could be private to start with
